packages/blueprints/gen-ai-chatbot/src/blueprint.ts

```diff
@@ -208,6 +214,17 @@ export class Blueprint extends ParentBlueprint {
       content: 'Log into your chatbot and use the bot console on the left to add custom data to the bot.',
     });
 
+    {
+      const models = Object.values(requiredModels);
+      const last = models.pop();
+      const modelNames = `${models.join(', ')}, and ${last}`;
+      new Issue(this, 'validate-model-access', {
```

Comment:
Is there a way we can do this check with a model invoke in the workflow?

Reply:
Feel free to merge and address in a follow up if you want

Reply:
yup, was going to address that in a separate PR. a little more involved so wanted to get this out first.

Reply:
x